x/distribution/spec/05_hooks.md

```diff
@@ -5,26 +5,44 @@ order: 5
 # Hooks
 
 ## Create or modify delegation distribution
- 
+
  - triggered-by: `staking.MsgDelegate`, `staking.MsgBeginRedelegate`, `staking.MsgUndelegate`
 
-The pool of a new delegator bond will be 0 for the height at which the bond was
-added, or the withdrawal has taken place. This is achieved by setting
-`DelegationDistInfo.WithdrawalHeight` to the height of the triggering transaction. 
+### Before
+
+The delegation rewards are withdrawn to the delegator's withdraw address.
+The rewards include the current period, but excludes the starting period.
+Then increment the validator period, which is used for calculating rewards.
+The latter is necessary because, due to the nature of the triggers, the validator's power and share distribution can have changed.
+The reference count for the delegator's starting period is decremented.
+
+### After
+
+The starting height of the delegation is set to the previous period.
+Because of the `Before`-hook, this is the last period for which the delegator was rewarded.
```

Comment:
```suggestion
Because of the `Before`-hook, this height is the last period for which the delegator was rewarded.
```

Reply:
let's make sure that "this" clarifies the subject, I _think_ we mean height (but check!)

Reply:
should be `period`, good catch